local/config/finna/Quria.ini.sample

```diff
@@ -0,0 +1,182 @@
+sort['AUTHOR ASCENDING'] = 'sort_author'
+sort['TITLE ASCENDING'] = 'sort_title'
+
+[cancelHolds]
+HMACKeys = id
+
+; Uncomment the following lines to enable password (PIN) change
+[changePassword]
+; PIN change parameters.
+minLength = 4
+maxLength = 4
+
+; Uncomment the following line to enable changePickupLocation
+[changePickupLocation]
+method = driver
+
+; Uncomment the following lines to enable updating of address information.
+; Choose a method on how to update the address information (email or url).
+; if you have patron quria you can use driver method.
+[updateAddress]
+method = driver
+; emailAddress is required for the email method
+;emailAddress =
+; Uncomment needsApproval and keep it as false when using driver and patron quria
+needsApproval = false
+; Uncomment the following lines to change which fields are included in the form
+; when using the email method. Format for the fields is label:profile_field
+; Default is address, zip, city and country. Email and phone are included unless
+; they have their own sections [updateEmail] or [updatePhone] defined. Also custom
+; fields can be defined.
+; When using patron quria and driver method, you need to
+; uncomment address, zip and city for updateAddress to work correctly via driver.
+fields[] = Address:address1
+fields[] = Zip:zip
+fields[] = City:city
+fields[] = Country:country
+;fields[] = Phone:phone
+;fields[] = Email:email
+;fields[] = Additional info:info
+; url is required for the url method. url can be a single address or an array of
+; language-specific addresses. First example is a single one:
+;url = http://address.of.change.form
+; Alternatively different links can be specified for each language:
+;url[fi] = http://address.of.change.form/finnish
+;url[sv] = http://address.of.change.form/swedish
+;url[en-gb] = http://address.of.change.form/english
+
+; Uncomment the following lines to enable updating of email address.
+; Choose a method on how to update the email address.
+; Only 'driver' is currently supported.
+[updateEmail]
+method = driver
+
+; Uncomment the following lines to enable updating of phone number.
+; Choose a method on how to update the phone number.
+; Only 'driver' is currently supported.
+[updatePhone]
+method = driver
+
+; Uncomment the following lines to enable update messaging settings
+; Choose a method on how to update the phone number (email, url or driver)
+[updateMessagingSettings]
+;method = email
+; emailAddress is required for the email method
+;emailAddress = "kirjasto@leppavirta.fi"
+; url is required for the url method. url can be a single address or an array of
+; language-specific addresses. First example is a single one:
+;url = "http://address.of.change.form"
+; Alternatively different links can be specified for each language:
+;url[fi] = http://address.of.change.form/finnish
+;url[sv] = http://address.of.change.form/swedish
+;url[en-gb] = http://address.of.change.form/english
+
+; valid value is 'driver'.
```

Comment:
tästä taitaa puuttua rivi 
`; Changing of loan history state is enabled if method parameter is set. Only`